### DISCUSSION TITLE
Recordings listed, but missing after database backup/restore

### DISCUSSION THREAD

**Discussion by qymm** [mcguirepr89/BirdNET-Pi#1218] [Q&A] [answered]
Hi there, was looking for some help.

I just installed BirdNET-Pi on a new Raspberry Pi 4B, upgrading from a Raspberry Pi 3B. New SD card too.

Fresh install, but I transferred the birds.db, the birdnet.conf, and the BirdSongs\Extracted\By_Date\ folders. I'm running into an issue where looking at `Recordings > By Date > 2024-02-02`, for instance, `American Crow` is listed, but clicking on it shows `No recordings were found.` The recordings are physically present in the directory, both .mp3 and .png, but are not listed in the webUI. What could I be missing?

**Comment by Nachtzuster** [comment]
wild shot: that old classic, permissions maybe? or ownership?

**Reply by qymm** [reply]
I'd believe it. I am very novice to Linux filesystems, would that be solved by running a recursive chown to user? Another user?

**Reply by Nachtzuster** [reply]
just running update from the tools menu does fix some of these thing, give that a try Tools->System->update

**Reply by qymm** [reply]
That's been done already, it did not have an effect.

I also just ran an `ls -g` on a set of soundfiles that are appearing, and a set of soundfiles that are not appearing, and both appear to have the same permissions/ownership:

Not appearing:
```-rw-r--r-- 1 $USER 96384 Feb  2 15:00 American_Crow-92-2024-02-02-birdnet-14_00_11.mp3```
Appearing:
```-rw-r--r-- 1 $USER 96384 Jun 20 05:58 American_Crow-55-2024-06-20-birdnet-05:58:16.mp3```

**Reply by qymm** [reply]
Oh my word, I just noticed the discrepancy. The filename was sanitized at some point, the colons were removed from the timestamp.

**Reply by Nachtzuster** [reply]
hmm, I did not see that one coming :smile:

**Reply by qymm** [reply, marked as answer]
I am running a regex rename in order to rename each of the files in the directory that match with the sanitized file format back to include colons in the timestamp.

This has solved the issue, at least for one date so far. Thanks for rubber ducking for me.

`find . -type f \( -name "*-*-*-birdnet-*_*.mp3" -o -name "*-*-*-birdnet-*_*.mp3.png" \) -exec rename 's/(.*-birdnet-\d{2})_(\d{2})_(\d{2})(\.mp3(?:\.png)?)$/$1:$2:$3$4/' {} +
`
